### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by psychemedia** [innovationOUtside/nbev3devsim#13]
The ev3dev sim UI quite possibly isn't very accessible.

Please add any accessibility issues you identify to this issue, ideally along with a suggestion about how to fix it, or at least a pointer to related accessibility guidelines.

**Comment by psychemedia** [comment]
- [ ] The `nb_tutor` display provides a low contrast display when reporting variable values. Related issue [here](https://github.com/lgpage/nbtutor/issues/37).

**Comment by psychemedia** [comment]
- [x] The simulator output window has a low contrast display when rendering error messages (by default, red on black); either change this b hardwiring in another colour (eg pink font) or provide a way of selecting high contrast theme/display. [Set in `main.css` via `#output > .error`]
